frontend/www/js/omegaup/components/login/Signup.vue

```diff
@@ -125,6 +130,57 @@ export default class Signup extends Vue {
   passwordConfirmation: string = '';
   recaptchaResponse: string = '';
 
+  mounted() {
+    const title = T.signUpFormInteractiveGuideTitle;
+
+    if (!this.hasVisitedSection) {
+      introJs()
+        .setOptions({
+          nextLabel: T.interactiveGuideNextButton,
+          prevLabel: T.interactiveGuidePreviousButton,
+          doneLabel: T.interactiveGuideDoneButton,
+          steps: [
+            {
+              title,
+              intro: T.signUpFormInteractiveGuideWelcome,
+            },
+            {
+              element: document.querySelector('.introjsusername'),
```

Comment:
En el archivo de declaración que encontré para `introjs` parece que `element` no permite valores `null`, y como la función de `querySelector` puede devolvernos dicho valor, acá podemos indicar el tipo que esperamos se regrese:
```suggestion
              element: document.querySelector('.introjsusername') as Element,
```

Esto mismo hay que aplicarlo en todos los items `element`